actionpack/CHANGELOG.md

```diff
@@ -1,18 +1,6 @@
-*   When multiple domains are specified for a cookie, a domain will now be
```

Comment:
I think you remove the other changelog entries. Can you add back?

Reply:
Apologies! Must have borked my rebase